calcom/synclinear.com
# Check repo against existing webhook instead of DB

# Summary

- Addresses [this comment](https://github.com/calcom/synclinear.com/issues/32#issuecomment-1331027596) from #32 
- Only show repo as "deployed" if webhook exists (previously checked DB for repo), in case of manual webhook deletion

## Test Plan

- Click "Connect GitHub". Repos should be listed.
- Select a synced repo. It should be "deployed".
- Select a non-synced repo. It should show the "Deploy button".

<img width="598" alt="image" src="https://user-images.githubusercontent.com/36117635/206515735-1a1383ea-045f-4e77-b0d1-61a73ed12382.png">


## Related Issues

Mitigates #32 and #33
